js/src/edit-free-campaign/index.js

```diff
@@ -102,8 +102,8 @@ export default function EditFreeCampaign() {
 	const [ settings, updateSettings ] = useState( savedSettings );
 
 	const {
+		hasFinishedResolution: hfrShippingRates,
```

Comment:
💅 

`hfr` felt a bit hard to understand for me. It took for me a while until I understood it was mean "hasFinishedResolution"

Reply:
@puntope , your comment here is same as @eason9487's comment in https://github.com/woocommerce/google-listings-and-ads/pull/1146#discussion_r768439360 (@eason9487 and I had an internal discussion in p1639476270030200-slack-C01E9LB4X61). I'm open to a better naming, do you have a suggestion? 🤔